### PR TITLE
Add currency exchange functionality

### LINKED ISSUE #62: Currency exchange

### PR DESCRIPTION
**Pull request by MonkeScripts**
Please take a look.
Not sure if this paradigm is what we need
Resolves #62

### CODE REVIEW

File: src/main/java/seedu/duke/CurrencyConversions.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//@@author MonkeScripts
+//all rates are relative to 1 SGD
+public enum CurrencyConversions {
+    USD("USD", 0.74),
+    RMB("RMB", 5.35),
+    EUR("EUR", 0.687),
+    JPY("JPY", 112.12),
+    AUD("AUD", 1.12),
+    MYR("MYR", 3.50),
+    SGD("USD", 1.00),
```
Comment (mukund1403) [inline src/main/java/seedu/duke/CurrencyConversions.java:11]: Are you sure USD to SGD is 1? https://www.google.com/search?q=usd+to+sgd&rlz=1C1KNTJ_enSG1055SG1055&oq=usd+to&gs_lcrp=EgZjaHJvbWUqCggAEAAYsQMYgAQyCggAEAAYsQMYgAQyBggBEEUYOTIKCAIQABixAxiABDIKCAMQABixAxiABDIKCAQQABixAxiABDIKCAUQABixAxiABDIKCAYQABixAxiABDIKCAcQABixAxiABDIKCAgQABixAxiABDIHCAkQABiPAqgCALACAA&sourceid=chrome&ie=UTF-8

File: src/main/java/seedu/duke/CurrencyConversions.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package seedu.duke;
+//@@author MonkeScripts
+//all rates are relative to 1 SGD
+public enum CurrencyConversions {
+    USD("USD", 0.74),
+    RMB("RMB", 5.35),
+    EUR("EUR", 0.687),
+    JPY("JPY", 112.12),
+    AUD("AUD", 1.12),
```
Comment (mukund1403) [inline src/main/java/seedu/duke/CurrencyConversions.java:9]: What if the user enters a currency that is not within this list? Can add exceptions for that in the future.

**Review by mukund1403 — CHANGES_REQUESTED** [review]
Overall looks good. Can consider the 2 points mentioned and also consider adding type.Epic to the issues.

**Review by Cohii2 — APPROVED** [review]
Good use of OOP and abstracting functionality of currency converter to new Money class!

**Pull request merged**